lib/web/mage/menu.js

```diff
@@ -461,7 +461,7 @@ define([
 
                 this.categoryLink = $('<a>')
                     .attr('href', categoryUrl)
-                    .text($.mage.__('All ') + category);
+                    .text($.mage.__('All') + ' ' + category);
```

Comment:
We must not use concatenation,, there should a placeholder within message, something like `'All %s'`

Reply:
There will be in issue - all lang packs already contains translation for "All", but none - for "All %s". I think it could be treated as backward incompatible change.

@joni-jones could you review it?

Reply:
@ihor-sviziev I don't know how it's organized but generally adding new translations is a normal maintenance process and has nothing to do with backward compatibility.